### DISCUSSION TITLE
DAC recommendations

### DISCUSSION THREAD

**Discussion by JoaquinSainz** [probonopd/MiniDexed#153] [Q&A] [answered]
hello guys I'm about to buy a DAC for this great project, I'm using the rasberry pi 3b. I have seen that many are using the Raspiaudio brand DACs, which one do you recommend for the audio quality, compatibility and price?

**Comment by probonopd** [comment, marked as answer]
Hello @JoaquinSainz.
Please see https://github.com/probonopd/MiniDexed/wiki/Hardware.
